simulator_stack/thruster_remap_g500/src/remap.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+import roslib;roslib.load_manifest('thruster_remap_g500')
+
+import rospy
+import kraken_msgs
+import geometry_msgs.msg as gs
+from kraken_msgs.msg._forceData6Thruster import forceData6Thruster
```

Comment:
ImportError: No module named msg._forceData6Thruster 

Facing this error when the roslaunch command is run.